Tests/AppTests/BuildIndexModelTests.swift

```diff
@@ -42,7 +42,18 @@ class BuildIndexModelTests: AppTestCase {
 
     func test_completedBuildCount() throws {
         let m = BuildIndex.Model.mock
-        XCTAssertEqual(m.completedBuildCount, 71)  // 72 total with one pending
+        // mock contains build for three Swift versions, 5.3, 5.4, 5.5
+        // each is set up the same:
+        // - 4 x .ok
+        // - 1 x .failed
+        // - 1 x .triggered
+        // - 1 x .timeout
+        // - 1 x .infrastructureError
+        // -> 6 completed per Swift version (4 x .ok + .failed + .timeout)
```

Comment:
This is a slight change: I've now added `.timeout` to the completed count, because it terms of processing we're done.